arenadata/Dockerfile

```diff
@@ -1,5 +1,17 @@
 FROM rockylinux:8.8 as base
 
+# Allow usage of custom compilers, however, gcc is the default
+ARG CC=gcc
+ARG CXX=g++
+ARG SKIP_UNITTESTS=false
+ARG USE_CLANG_ANALYZER=false
+
+RUN \
+    if [ "$USE_CLANG_ANALYZER" = true ]; then \
+        export CFLAGS="-fsyntax-only --analyze"; \
+        export CXXFLAGS="-fsyntax-only --analyze"; \
```

Comment:
I run the command from arenadata/readme.md and there is no "-fsyntax-only --analyze" substring in the configure output
```
configure: using CFLAGS=-Wall -Wmissing-prototypes -Wpointer-arith -Wimplicit-fallthrough -Wvla -Wno-deprecated-declarations -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-unused-but-set-variable -Wno-format-truncation -Wno-stringop-truncation -g -O3 -fno-omit-frame-pointer -g  -Werror
configure: using CXXFLAGS=-Wall -Wpointer-arith -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -g -O3 -Werror
```

Reply:
These flags are redundant, so I removed them